lib/bindings/python/rust/context.rs

```diff
@@ -15,11 +16,29 @@ use std::sync::Arc;
 #[pyclass]
 pub struct Context {
     inner: Arc<dyn AsyncEngineContext>,
+    trace_context: Option<DistributedTraceContext>,
 }
 
 impl Context {
     pub fn new(inner: Arc<dyn AsyncEngineContext>) -> Self {
-        Self { inner }
+        Self {
+            inner,
+            trace_context: None,
+        }
+    }
+
+    // Create context with trace information
+    #[must_use]
```

Comment:
not sure if the `#[must_use]` needs to be explicitly marked, because if the caller is ignoring the returned `Context`, then this method becomes a no-ops and consumes the `DistributedTraceContext`?

Reply:
removed, since this is combined with the `new` constructor